### ISSUE TITLE
ELB DNS name timing out after deployment

### ISSUE THREAD

**Issue by benazir0** [onnela-lab/beiwe-backend#43]
I have successfully deployed the backend with some minor tweaks on getting the recent postgres engine name and version. However, the DNS name on the load balancer is not leading to anywhere when I try to access it via web browser. The Elastic Beanstalk health is ok and I don't see any error message. Please advise how to resolve this issue.

**Comment by benazir0** [comment]
Solved it. I didn't notice SSL code was uncommented on the config file, therefore, SSL setup was required. After setting up SSL and allowing ssl on security group, it's no longer timing out. However, I can't login with admin, admin credentials. Do I need to complete data-pipeline setup as well? Where can I reset the password?

**Comment by benazir0** [comment]
Found it on the management folder. I'm closing it.

**Comment by abhineethreddyk** [comment]
@benazir0 I've completed the deployment and I'm able to access the login page. However, I'm unable to log-in using admin-admin credentials. What would I need to do in the management folder to create/use admin user?

**Comment by biblicabeebli** [comment]
The creation of the admin user is built into the database schema setup operations during that first deployment.  It cannot be bypassed or avoided, and it is guaranteed to occur only once.

If you are not able to resolve this please create a new issue so we can address it.
